### PR TITLE
Zonal Electric Heating Setpoints

### PR DESCRIPTION
**Pull request by afontani**
## Pull Request Description

**Companion PR:** resstock-estimation ([#96](https://github.com/NREL/resstock-estimation/pull/96))

Adds zonal electric heating equipment as a dependency for heating setpoint-related tsvs (`Heating Setpoint`, `Heating Setpoint Offset Period`, `Heating Setpoint Offset Magnitude` and `Heating Setpoint Has Offset`).  Zonal electric heating includes "Built-In Electric Units" and "Portable Electric Heaters" in RECS 2009. Additionally, all weekend daytime heating and cooling setpoint offsets are removed

## Checklist

Not all may apply:

- [ ] Unit tests have been added or updated
- [x] All rake tasks have been run, and pass
- [ ] Documentation has been modified appropriately
- [ ] Any new options are added to `project_testing`
- [ ] `project_testing` runs without any failures
- [ ] No unexpected regression test changes
- [x] All tests are passing
- [x] The [changelog](https://github.com/NREL/resstock/blob/master/CHANGELOG.md) has been updated appropriately
- [x] This branch is up-to-date with develop

For more information on how to perform these checklist items, see the documentation's [Advanced Tutorial](https://resstock.readthedocs.io/en/latest/advanced_tutorial/index.html).

### CODE REVIEW

**Review by aspeake — APPROVED** [review]

File: resources/options_lookup.tsv [hunk 1]
```diff
@@ -938,17 +940,17 @@ Cooling Setpoint Offset Magnitude	2F	ResidentialHVACCoolingSetpoints	weekday_off
...
-Cooling Setpoint Offset Period	Day Setup -3h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -4h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0"																																											
-Cooling Setpoint Offset Period	Day Setup -5h	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0,0,0,0,0,0"																																											
+Cooling Setpoint Offset Period	Day Setup	ResidentialHVACCoolingSetpoints	"weekday_offset_schedule=0,0,0,0,0,0,0,0,0,1,1,1,1,1,1,1,1,0,0,0,0,0,0,0"	"weekend_offset_schedule=0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0"
```
Comment (joseph-robertson) [inline resources/options_lookup.tsv:943]: @afontani Why this change but not for Night Setup?

Comment (aspeake) [reply resources/options_lookup.tsv:943]: @joseph-robertson It is assumed that a day setback is usually applied during unoccupied times, which we can't assume during weekends. However, if there is a night setback that would be the desired temperature while asleep, which probably would not be affected by weekday/weekend days.
